src/Altinn.App.Core/EFormidling/Implementation/DefaultEFormidlingService.cs

```diff
@@ -66,7 +65,7 @@ public class DefaultEFormidlingService : IEFormidlingService
     public async Task SendEFormidlingShipment(Instance instance)
     {
         if (_eFormidlingClient == null || _tokenGenerator == null || _eFormidlingMetadata == null ||
-            _appSettings == null || _platformSettings == null)
+            _appSettings == null || _platformSettings == null || _userTokenProvider == null)
```

Comment:
I think this null check of _userTokenProvider can be removed?

The depdendency container in asp net will throw an error if it fails to construct and inject this dependency, since it's not optional. If you still want to null check it in case the class is instantiated, I would throw in the constructor instead.

Reply:
You are right, I will remove the null check for _userTokenProvider